build/build_binary.sh

```diff
@@ -82,7 +82,8 @@ EOM
 }
 
 writeConfig
-cp ${PROJECT_DIR}/LICENSE ${PROJECT_DIR}/NOTICE ${release_dir}
+cp ${PROJECT_DIR}/licenses/LICENSE ${PROJECT_DIR}/licenses/NOTICE ${release_dir}
+cp -r ${PROJECT_DIR}/licenses ${release_dir}
```

Comment:
So the licenses directory is still have the copied the NOTICE and LICENSE file, we need to clean them up.

Reply:
@WillemJiang  Do you mean that we just need to keep the `cp -r ${PROJECT_DIR}/licenses ${release_dir}`?
This is the build_binary.sh. I checked the servicecomb-mesher binary package, which has a LICENSE file  and a licenses directory in the root directory.

Reply:
@WillemJiang  Do you mean that we just need to keep the `cp -r ${PROJECT_DIR}/licenses ${release_dir}`?
This is the build_binary.sh. I checked the servicecomb-mesher binary package, which has a LICENSE file  and a licenses directory in the root directory. 
@WillemJiang  Do you mean that we just need to keep the `cp -r ${PROJECT_DIR}/licenses ${release_dir}`?
This is the build_binary.sh. I checked the servicecomb-mesher binary package, which has a LICENSE file  and a licenses directory in the root directory.

Reply:
yes, we just need to keep the LICENSE and NOTICE file in the root directory of the artifact.